component-samples/demo/README.MD

```diff
@@ -293,6 +317,28 @@ During SVI Transfer, observe the column `MEM USAGE / LIMIT` for respective conta
 If the failure is observed when MEM USAGE reaches the LIMIT, it is due to the memory limit we have allocated for the container.
 To resolve this, edit fields `mem_limit` and `mem_reservation` in the respective docker-compose.yml according to the needs and restart the container.
 
+# Connecting to external resources during SVI
+
+When connecting to the external signed resource, the application should be using the default java ca-certs instead of the custom truststore. The following properties in `service.yml` ought to be commented out to pick the default java ca-certs.
```

Comment:
ought is good. let us use need instead.
Actually we need to start following the standards around SHOULD MUST and all that .. maybe in a future release

Reply:
Done.